components/indexer.py

```diff
@@ -1,33 +1,47 @@
 from magicbot import feedback
+import ctre
+import wpilib
 
 
 class Indexer:
     indexer_motors: list
     indexer_switches: list
+    piston_switch: wpilib.DigitalInput
+    injector_switch: wpilib.DigitalInput
+    injector_master_motor: ctre.WPI_TalonSRX
+    injector_slave_motor: ctre.WPI_TalonSRX
+
+    def setup(self):
+        for motor in self.indexer_motors:
+            motor.setInverted(True)
+
+        self.speed = 0.2
```

Comment:
Perhaps it would be better to have two global constant speeds for the indexer and injector? right now "speed" seems like a bad name anyway